README.md

```diff
@@ -1,2 +1,46 @@
 # codejam-virtual-keyboard
 Virtual keyboard RSS task
+
```

Comment:
Information about task isn't needed here. It would be nice if you was added a short instruction how I can run project locally.